src/components/DeckMechanisms/index.js

```diff
@@ -16,13 +27,13 @@ export default class DeckMechanisms extends React.Component {
   constructor(props) {
     super(props)
 
-    this.state = {
+    this.DEFAULT_STATE = {
```

Comment:
You should move this out of the class itself.

```js
const DEFAULT_STATE = { … }

export default class DeckMechanisms extends React.Component {
  constructor(props) {
    super(props)
    
    this.state = { ...DEFAULT_STATE }
  }
}
```

Reply:
That's what I initially did, but I needed props to define DEFAULT_STATE

Reply:
You could make it a function then.
```js
const getDefaultProps = () => ({ … })
```

Reply:
I tried to do so, not sure this is what you meant